### ISSUE TITLE
Can not completely block javascript at accounts.google.com

### ISSUE THREAD

**Issue by snowbound** [gorhill/uMatrix#793]
I want to force gmail to use their older style login screen which logs you out when Google Chrome is closed. Google new login screens for Gmail keeps you logged in unless you use Gmail logout option via its pulldown menu or use incognito mode. If I do NOT have uMatrix enabled and follow the following below steps. I see the older style login screen for Gmail. 

However if I enable uMatrix the below steps do not work. This is even if I have scripts for accounts.google.com block is red.

Why is this happening?
-------------------------------------------------

However, for your own computer, try this:

FIRST remove any/all Gmail accounts from the sign in page. 
To do this, use  "remove account" at the bottom of the sign in page, then click the X to remove the account.
If there is only one account listed or you don't see "remove account" click the drop-down box to the right of your user name first to see "remove account".
Once all accounts have been removed from the sign in page, then follow these steps:

Click the 3 dots at the upper right corner of the Chrome Page
Select Settings
Scroll to the bottom and click "Show Advanced Settings"
In the Privacy section, click "Content settings"
Under Cookies, be sure "Allow local data to be set (recommended)" is filled in.
Under JavaScript, be sure "Allow all sites to run JavaScript (recommended) is filled in.
Under JavaScript, click "Manage exceptions"
Under Hostname pattern, type https://accounts.google.com
Under Behaviour, select "Block"
Click "Finished" (takes you back to 'content settings')
Click "Finished" (takes you back to 'settings')

Close Chrome.

Open Chrome.
Open Gmail. You should see the old sign in page. Enter your username.
UNCHECK "Stay signed in" and enter your password.
Close Chrome.
Open Chrome.
You should now be back to the old sign in page and must enter your username and password to access your account.

**Comment by gorhill** [comment]
Javascript is correctly blocked, just look at the console for the page.

The issue is duplicate of #319: with uMatrix the `noscript` tag is not rendered. If blocking javascript using the browser's own blocker, the `noscript` tag is rendered and it will cause the page to be redirected to the old account login.

Marking as duplicate of #319. I will fix this by simply preventing uMatrix from overtaking the control of the browser's javascript setting. I did remove this in the past, but I put it back because back then I had problem to just say "no" (someone reported a case where he did not realize javascript was blocked by the browser and complained that uMatrix was not unblocking execution of javascript when creating an allow rule).

**Comment by snowbound** [comment]
The thing is that if javascript is enabled in Chrome it works fine IF UMatrix extension is not  enabled in Chrome. If I enable the extension even if scripts for accounts.google.com are enable aka green in UMatrix the old login screen never is displayed. 

If I want to see the older Gmail login I have to stop using UMatrix.
